integration-tests/buildtask-sdk/dependency/AssemblyInfo.cs

```diff
@@ -0,0 +1,2 @@
+﻿//[assembly: System.Runtime.CompilerServices.InternalsVisibleTo("buildtask-sdk")]
```

Comment:
Any reason to leave this first line, commented like this?

Reply:
No, will remove.